app/mailers/candidate_mailer.rb

```diff
@@ -60,15 +60,17 @@ def new_interview(application_choice, interview)
     )
   end
 
-  def interview_updated(application_choice, interview)
+  def interview_updated(application_choice, interview, old_course)
     @application_form = application_choice.application_form
     @interview = interview
     @provider_name = interview.provider.name
     @course_name_and_code = application_choice.current_course_option.course.name_and_code
+    @old_course_name_and_code = old_course.course.name_and_code
```

Comment:
We should stick to the convention of variables reflecting the underlying type
eg. `old_course_option.course.name_and_code` as the current code implies a course belongs to a course to the uninitiated.
Would also go with `previous_course_option` as it's slightly more accurate.